src/app/tasks/project-tasks-list/project-tasks-list.component.html

```diff
@@ -0,0 +1,18 @@
+<ul class="project-tasks-list">
+  <div *ngIf="project.tasks.length > 0" class="group-set-tasks clearfix">
+    <li *ngFor="let item of project.tasks" class="col-xs-4 col-lg-3">
+      <button
+        mat-raised-button
+        matTooltip="{{ taskDefinition(item).name }}:{{ item.status_txt() }}"
+        matTooltipPosition="below"
+        matTooltipShowDelay="200"
+        class="col-xs btn task-status {{ statusClass(item.status) }}"
+        (click)="onClickItem(item, 'Task Button List')"
+        style="width: 100%"
+      >
+        <i class="fa fa-eye" *ngIf="item.similar_to_count - item.similar_to_dismissed_count > 0"></i>
+        {{ taskText(item) }}
```

Comment:
If you do decide to display the grade and rating values in the tooltip, you could remove the `taskText` function entirely, and have `item.definition.abbreviation` instead. 

You could then have a function to get the tooltip label, which could push a string to an array for the grade value if it's graded, another string for the rating if it's rated, then join the array by a space and return it.